diff --git a/node_modules/expo-dev-launcher/expo-dev-launcher-gradle-plugin/build.gradle.kts b/node_modules/expo-dev-launcher/expo-dev-launcher-gradle-plugin/build.gradle.kts
index bbf752e..f5e38e1 100644
--- a/node_modules/expo-dev-launcher/expo-dev-launcher-gradle-plugin/build.gradle.kts
+++ b/node_modules/expo-dev-launcher/expo-dev-launcher-gradle-plugin/build.gradle.kts
@@ -1,33 +1,26 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
 plugins {
-  kotlin("jvm") version "1.9.23"
+  kotlin("jvm") version "2.1.20"
   id("java-gradle-plugin")
 }
-
 repositories {
   google()
   mavenCentral()
 }
-
 dependencies {
   implementation(gradleApi())
   compileOnly("com.android.tools.build:gradle:8.2.1")
 }
-
 java {
   sourceCompatibility = JavaVersion.VERSION_11
   targetCompatibility = JavaVersion.VERSION_11
 }
-
 tasks.withType<KotlinCompile> {
   kotlinOptions {
     jvmTarget = JavaVersion.VERSION_11.toString()
   }
 }
-
 group = "expo.modules"
-
 gradlePlugin {
   plugins {
     register("expoDevLauncherPlugin") {
